issue_11/script.js

```diff
@@ -0,0 +1,61 @@
+function startLoading() {
+    const loadingImg = document.createElement('img');
+    loadingImg.src = 'loading-circle.gif';
+    loadingImg.alt = 'ローディング画像';
+    document.getElementById('js-loading').appendChild(loadingImg);
+}
+
+function stopLoading() {
+    document.getElementById('js-loading').remove();
+}
+
+function createLists(resolvedData) {
+    const ul = document.getElementById('js-ul');
+    const fragment = document.createDocumentFragment();
+    for (const item of resolvedData) {
+        const li = document.createElement('li');
+        const a = document.createElement('a');
+        const img = document.createElement('img');
+
+        a.href = item.to;
+        a.textContent = item.text;
+        img.src = item.img;
+        img.alt = item.alt;
+
+        fragment.appendChild(li).appendChild(a).appendChild(img);
+    }
+    ul.appendChild(fragment);
+}
+
+
+async function callJson() {
+    const url = 'https://api.json-generator.com/templates/NluaaELSLhVe/data?access_token=ambjpfxjl00e50wa639kwndq1ofq3iuykdrv98ge';
+    const response = await fetch(url);
+    const result = await response.json();
+    return result.data;
+}
+
+function getJsonData() {
+    return new Promise((resolve) => {
+        setTimeout(() => {
+            resolve(callJson());
+        }, 3000);
+    });
+}
```

Comment:
Q
Why did you think this part was necessary?

Reply:
Thank you for comments.
Since the assignment description says "display the same as last time," I thought the retrieved json data needed to be resolved after 3 seconds.

Reply:
This task means to get a value from the API and display it the same as last time, so the technical specifications are different. The solution after the last three seconds is the same interpretation of the time it takes to fetch this time. So I don't think promise is necessary for this assignment.

Reply:
I see. I understand!
I have removed the relevant function.
https://github.com/yoshikawa-akr/js-lessons/pull/11/commits/58b1fedf245e0f6582bcf9e0ec6ff5847a8f8ffe